packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -30,26 +29,25 @@ export interface IJSONFormatProps extends ITruncatedFormatProps {
     stringify?: (obj: any) => string;
 }
 
-@PureRender
```

Comment:
> because column resize doesn't directly result in props or state changes

sounds like we should fix that instead. the current solution is a band-aid